docs/source/conf.py.in

```diff
@@ -115,9 +107,6 @@
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
-# If true, `todo` and `todoList` produce output, else they produce nothing.
-todo_include_todos = True
```

Comment:
Is the idea not to have our TODOs show on the public webpage? I mean, it seems to have no effect either way (Doxygen TODOs still don't get included), but what's the rationale here?

Reply:
I though the todo extension was disabled, my bad here. I'll correct it.